client/my-sites/site-logs/main.tsx

```diff
@@ -42,6 +53,8 @@ export function SiteLogs() {
 		.join( '\n' );
 
 	const titleHeader = __( 'Site Logs' );
+	const tabName = new URL( window.location.href ).searchParams.get( 'logType' ) ?? 'php';
```

Comment:
Just a nitpick, but I'm a fan of using only lower case in the URL. Maybe `?log-type=php`?